test/e2e/performanceprofile/functests/utils/nodes/nodes.go

```diff
@@ -299,6 +300,37 @@ func GetNumaNodes(node *corev1.Node) (map[int][]int, error) {
 	return numaCpus, err
 }
 
+// GetCoreSiblings returns the siblings of core per numa node
+func GetCoreSiblings(node *corev1.Node) (map[int]map[int][]int, error) {
+	lscpuCmd := []string{"lscpu", "-e=node,core,cpu", "-J"}
+	out, err := ExecCommandOnNode(lscpuCmd, node)
+	var result NumaNodes
+	var numaNode, core, cpu int
+	coreSibilings := make(map[int]map[int][]int)
```

Comment:
maybe simpler something that this:
`// GetCoreSiblings returns the siblings of core per numa node
func GetCoreSiblings(node *corev1.Node) ([][]int, error) {
	lscpuCmd := []string{"lscpu", "-e=node,core,cpu", "-J"}
	out, err := ExecCommandOnNode(lscpuCmd, node)
	if err != nil {
		return nil, err
	}
	var result NumaNodes
	coreSiblings := make([][]int, len(result.Cpus)/2)
	err = json.Unmarshal([]byte(out), &result)
	for _, value := range result.Cpus {
		var core, cpu int
		if core, err = strconv.Atoi(value.Core); err != nil {
			break
		}
		if cpu, err = strconv.Atoi(value.CPU); err != nil {
			break
		}
		if core != cpu {
			coreSiblings = append(coreSiblings, []int{core, cpu})
		}
	}
	return coreSiblings, err
}`

not tested, but the idea its not use that complicated map

Reply:
Yeah the map is complicated because we need Per numa node, per core siblings . 
So we get output like this:

```
map[0:map[0:[0 40] 10:[10 50] 12:[12 52] 14:[14 54] 16:[16 56]....
map[1:[1 41] 11:[11 51] 13:[13 53] 15:[15 55] 17:[17 57] 19:[19 59] ...
```

Reply:
Ok, I don't like so much the complexity but if it has to be like that...